dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBMDOrchestratorImpl.java

```diff
@@ -1342,32 +1344,43 @@ protected void processZOSMetadata(BeanMetaData bmd) {
         if (isTraceOn && tc.isEntryEnabled())
             Tr.entry(tc, "processZOSMetadata : " + bmd);
 
-        if (EJSPlatformHelper.isZOS()) {
-            bmd.m_syncToOSThreadValue = getSyncToOSThreadSetting(ContainerConfigConstants.syncToOSThreadSetting, bmd);
+        if (isZOS()) {
```

Comment:
What is wrong with using EJSPlatformHelper.isZOS()?

Reply:
Fixed as you suggest to use the ContainerProperties